### PR TITLE
[README] Create a package.json section

### PR DESCRIPTION
**Pull request by jablko**
Create a `package.json` section for the bot to link to, [to explain the required form](https://github.com/DefinitelyTyped/dt-mergebot/blob/25e165f39fba6d741bc4e9339f825c60d8d6fb28/src/pr-info.ts#L383).

Put sections in the following order:
- Edit an existing package
- Create a new package
- Common mistakes
- Removing a package
- Running Tests
- \<my package>-tests.ts
- Linter: `tslint.json`
- package.json

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -314,11 +297,42 @@ f("one");
...
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    // This package uses the Function type, and it will take effort to fix.
+    "ban-types": false
+  }
+}
+```
+
+(To indicate that a lint rule truly does not apply, use `// tslint:disable rule-name` or better, `//tslint:disable-next-line rule-name`.)
```
Comment (peterblazejewicz) [inline README.md:322]:
```suggestion
(To indicate that a lint rule truly does not apply, use `// tslint:disable rule-name` or better, `// tslint:disable-next-line rule-name`.)
```

File: README.pt.md [hunk 2]
```diff
@@ -314,11 +298,41 @@ f("um");
...
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        // Este pacote usa o tipo Function, e vai dar trabalho para arrumar.
+        "ban-types": false
+    }
+}
+```
+
+(Para indicar que uma regra de lint de fato não se aplica, use `// tslint:disable nome-da-regra` ou até mesmo, `//tslint:disable-next-line nome-da-regra`.)
```
Comment (peterblazejewicz) [inline README.pt.md:322]:
```suggestion
(Para indicar que uma regra de lint de fato não se aplica, use `// tslint:disable nome-da-regra` ou até mesmo, `// tslint:disable-next-line nome-da-regra`.)

```

File: README.ru.md [hunk 3]
```diff
@@ -304,11 +287,36 @@ f('one');
...
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        // This package uses the Function type, and it will take effort to fix.
+        "ban-types": false
+    }
+}
+```
+
+(Чтобы указать, что правило lint действительно не применяется, используйте `// tslint:disable rule-name` или лучше, `//tslint:disable-next-line rule-name`.)
```
Comment (peterblazejewicz) [inline README.ru.md:312]:
```suggestion
(Чтобы указать, что правило lint действительно не применяется, используйте `// tslint:disable rule-name` или лучше, `// tslint:disable-next-line rule-name`.)
```

File: README.cn.md [hunk 4]
```diff
@@ -309,10 +293,41 @@ f("one");
...
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        // This package uses the Function type, and it will take effort to fix.
+        "ban-types": false
+    }
+}
+```
+
+(若要使某个 lint 规则不生效，可以使用 `// tslint:disable rule-name`，当然使用 `//tslint:disable-next-line rule-name` 更好。)
```
Comment (peterblazejewicz) [inline README.cn.md:316]:
```suggestion
(若要使某个 lint 规则不生效，可以使用 `// tslint:disable rule-name`，当然使用 `// tslint:disable-next-line rule-name` 更好。)
```

File: README.es.md [hunk 5]
```diff
@@ -280,10 +268,29 @@ f("one");
...
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        // This package uses the Function type, and it will take effort to fix.
+        "ban-types": false
+    }
+}
+```
+
+(Para indicar que la regla lint realmente no es utilizada, usa `// tslint:disable rule-name` o mejor, `//tslint:disable-next-line rule-name`.)
```
Comment (peterblazejewicz) [inline README.es.md:286]:
```suggestion
(Para indicar que la regla lint realmente no es utilizada, usa `// tslint:disable rule-name` o mejor, `// tslint:disable-next-line rule-name`.)
```

File: README.ko.md [hunk 6]
```diff
@@ -304,11 +293,29 @@ f("one");
...
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        // 이 패키지는 Function 형을 사용하고 있으며, 고치는 게 쉽지 않다.
+        "ban-types": false
+    }
+}
+```
+
+(린트 규칙(Lint rule)이 절대로 적용되서는 안되는 경우에는, `// tslint:disable rule-name` 나 `//tslint:disable-next-line rule-name` 를 사용하는 것이 좋습니다. 후자가 더 나은 방식입니다.)
```
Comment (peterblazejewicz) [inline README.ko.md:311]:
```suggestion
(린트 규칙(Lint rule)이 절대로 적용되서는 안되는 경우에는, `// tslint:disable rule-name` 나 `// tslint:disable-next-line rule-name` 를 사용하는 것이 좋습니다. 후자가 더 나은 방식입니다.)
```

**Review by peterblazejewicz — APPROVED** [review]
I hope you're know what you're doing 👍

**Comment by sheetalkamat** [comment]
Waiting on @orta to review multi language changes.

File: README.md [hunk 7]
```diff
@@ -225,7 +226,7 @@ You can remove it by running `npm run not-needed — typingsPackageName asOfVers
 
 Any other packages in Definitely Typed that referenced the deleted package should be updated to reference the bundled types.
 You can get this list by looking at the errors from `npm run test-all`.
-To fix the errors, add a `package.json` with `"dependencies": { "foo": "x.y.z" }`.
+To fix the errors, [add a `package.json`](#packagejson) with `"dependencies": { "foo": "x.y.z" }`.
```
Comment (amcasey) [inline README.md:229]: Should the example and notes below be moved to the package.json section?

**Review by amcasey — COMMENTED** [review]
So this basically just promotes an FAQ entry to a section and then reorders a bunch of things?

File: README.md (hunk 7 above)
Comment (jablko) [reply README.md:229]: I thought about that and am happy to go either way. My thoughts were
1. I decided it belonged with "Removing a package". The steps in that section are less followable without it and it doesn't contribute much to the new "package.json" section. Both sections explain `allowedPackageJsonDependencies.txt`.
2. Not having a strong indication in favor of editing it, I opted to minimize this change. Those sections can be reorganized or combined subsequently, and promoting the FAQ entry is neutral at worst, I think?

**Comment by jablko** [comment]
> So this basically just promotes an FAQ entry to a section and then reorders a bunch of things?

Yup exactly.

File: README.md (hunk 7 above)
Comment (amcasey) [reply README.md:229]: I can live with that.  My thinking was mostly that I would look for package.json info in the corresponding section (which is not the case for an FAQ entry) and might be annoyed to discover that it was elsewhere.  However, as you point out, the sections seem to have different goals and the example makes sense here.

**Review by amcasey — APPROVED** [review]
The English changes LGTM.

**Comment by orta** [comment]
Looks good to me too, it's mainly just shuffling the translations around rather than language specific improvements 👍🏻

**Pull request merged**